tasks/debian.yml

```diff
@@ -1,13 +1,37 @@
 ---
 
-- name: python3 venv | install python3
+- name: install software-properties-common
+  ansible.builtin.apt:
+    name: software-properties-common
+    state: present
+    update_cache: yes
+
+- name: Add deadsnakes PPA
+  ansible.builtin.apt_repository:
+    repo: ppa:deadsnakes/ppa
+    state: present
+
+- name: apt cache update
+  ansible.builtin.apt:
+      update_cache: yes
+
+- name: python3.12 venv | install python3
```

Comment:
We discussed adding an option to specify the Python version 
similar to https://github.com/ome/ansible-role-java